src/test/java/com/bootexample4/RoostTest/refundMessagePostTest.java

```diff
@@ -0,0 +1,86 @@
+// ********RoostGPT********
+/*
+Test generated by RoostGPT for test rest-assured-loader-testing using AI Type Open AI and AI Model gpt-4
+
+Test generated for /icd/s2s/exchange/refundMessage_post for http method type POST in rest-assured framework
+
+RoostTestHash=19003958c4
+
+
+*/
+
+// ********RoostGPT********
+package com.bootexample4.RoostTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import static io.restassured.RestAssured.given;
+import static org.junit.Assert.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import org.hamcrest.MatcherAssert;
+import static org.hamcrest.Matchers.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class refundMessagePostTest {
+
+    Map<String, String> envs = new HashMap<>();
+
+
+    @Before
+    public void setUp() {
+      Dataloader envData = new Dataloader();
```

Comment:
```suggestion
      TestDataloader dataloader = new TestDataloader();
```
In template, declaration should be like this. so that class functions can be called like
`dataloader.FromEnv(), dataloader.FromProperties(), dataloader.FromCSV`

Reply:
change made.